patches/chrome-browser-resources-settings-route.js.patch

```diff
@@ -1,8 +1,24 @@
 diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
-index 3099fa50bcfc1e2ba442910650de4fcdd871fdca..7d888a7eb93889384684ae4b3862e1efb5596e82 100644
+index 3099fa50bcfc1e2ba442910650de4fcdd871fdca..13d5ec1351490f15bb2ffc27945a930c4739eb27 100644
 --- a/chrome/browser/resources/settings/route.js
 +++ b/chrome/browser/resources/settings/route.js
-@@ -333,9 +333,12 @@ cr.define('settings', function() {
+@@ -95,6 +95,7 @@
+  *   SITE_SETTINGS_UNSANDBOXED_PLUGINS: (undefined|!settings.Route),
+  *   SITE_SETTINGS_USB_DEVICES: (undefined|!settings.Route),
+  *   SITE_SETTINGS_ZOOM_LEVELS: (undefined|!settings.Route),
++ *   SHIELDS: (undefined|!settings.Route),
```

Comment:
Seems unnecessary to patch this line because it was just for @typedef, we didn't add one when we patched for adding r.SITE_SETTINGS_AUTOPLAY.

Reply:
++ on this - I think we can remove that line